features/git-town-hack/hack_push_flag_true.feature

```diff
@@ -0,0 +1,28 @@
+Feature: git town-hack: push branch to remote upon creation
+
+  As a developer starting work on a new feature
+  I want to be able to configure git-hack to push my branch to the remote repo
+  So that ...
```

Comment:
@kevgo could you update / finish off this user value? The idea was originally discussed in #664